nikola/data/themes/base/templates/feeds_translations_helper.tmpl

```diff
@@ -1,91 +1,97 @@
 ## -*- coding: utf-8 -*-
 
-## Handles both feeds and translations
-<%def name="head(classification=None)">
-    % if rss_link:
+<%def name="_append_language(language)">${ " (" + language + ")" if len(translations) > 1 else "" }</%def>
```

Comment:
I modified jinjify in a340ffea1f7a6ab1a2e439b5b8042b73c9d3425e so it can handle this.